# Example of a detection message
builtin_interfaces/Time stamp
string object_name
float32 x
float32 y
float32 z
